### PR TITLE
chore: clarify personal access token scopes

### PR DESCRIPTION
**Pull request by sweatybridge**
## What kind of change does this PR introduce?

docs

## What is the new behavior?

Clarify that PATs can perform the same actions as users on dashboard. 

## Additional context

Screenshot 2023-06-22 at 4 40 29 PM

### CODE REVIEW

File: studio/pages/account/tokens.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const UserAccessTokens: NextPageWithLayout = () => {
       <div className="flex items-center justify-between">
         <FormHeader
           title="Access Tokens"
-          description="Personal access tokens can be used with our management API or the Supabase CLI"
+          description="Personal access tokens can be used with our management API or the Supabase CLI. They inherit the same scopes as your actions on the dashboard."
```
Comment (MildTomato) [inline studio/pages/account/tokens.tsx:23]: > They inherit the same scopes as your actions on the dashboard

What are you trying to say? this sounds vague

Comment (MildTomato) [reply studio/pages/account/tokens.tsx:23]: If its "The personal access tokens will have the same permissions you have" can we not just say that.

Comment (sweatybridge) [reply studio/pages/account/tokens.tsx:23]: Discussed offline. Will be making this into an alert box so we have more space to explain in detail.

**Review by MildTomato — APPROVED** [review]

File: studio/pages/account/tokens.tsx (hunk 1 above)
Comment (MildTomato) [reply studio/pages/account/tokens.tsx:23]: It's not that bad though, i've approved anyway. keep things moving

Comment (sweatybridge) [reply studio/pages/account/tokens.tsx]: Ok will merge first and improve later if users find it unclear.

**Pull request merged**